template/en/default/setup/strings.txt.pl

```diff
@@ -54,7 +54,7 @@ END
 to continue.
 END
     cpanfile_created => "##file## created",
-    cpan_bugzilla_home => 
```

Comment:
I had a hard time finding the actual code changes due to the sheer volume of unrelated whitespace fixes in this pull request. Please consider separating them into their own whitespace-only commit and/or pull request in the future.